modules/table/src/main/java/org/apache/ignite/internal/table/distributed/command/CommandUtils.java

```diff
@@ -138,6 +144,88 @@ public static void readRows(byte[] bytes, Consumer<BinaryRow> consumer) {
         }
     }
 
+    /**
+     * Writes a row map to byte array.
+     *
+     * @param map Map a row id to the binary row.
+     * @return Array of bytes.
+     */
+    public static byte[] writeRowMap(Map<RowId, BinaryRow> map) {
```

Comment:
For the consistency purposes I'd favor overloading rowsToBytes and rowsToBytes, meaning that should be fine to rename weoreRowMap and readRowMap methods. Up to you though.

Reply:
Rename writeRowMap to rowMapToBytes